CMakeLists.txt

```diff
@@ -33,6 +33,10 @@ if(CMAKE_CROSSCOMPILING)
     cmake_minimum_required(VERSION 3.3) # Version introducing CROSSCOMPILING_EMULATOR
 endif()
 
+if(WIN32 AND "${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
```

Comment:
The `CMAKE_DEBUG_POSTFIX` variable only preseeds the `Debug` build type suffix, so there is no need to add that to the condition.

https://cmake.org/cmake/help/v2.8.5/cmake.html#variable:CMAKE_DEBUG_POSTFIX